config/default.json

```diff
@@ -0,0 +1,52 @@
+{
+    "robotYarpPortPrefix": "/icubSim",
+    "telemVizServer": {
+        "host": "localhost",
+        "port": 8081
+    },
+    "consoleServer": {
+        "host": "localhost",
+        "port": 3000
+    },
+    "openmctStaticServer": {
+        "host": "localhost",
+        "port": 8080
+    },
+    "portInConfig": {
+        "sens.imu": {
```

Comment:
Is the prefix ``sens`` always needed?

Reply:
It was an initial choice to have this prefix in case we added in https://github.com/ami-iit/yarp-openmct/blob/f2144093b7a6ea2b9fce1d0db7aab9a5bd33a63a/iCubTelemVizServer/icubtelemetry.js#L21 other entries not directly associated to a sensor output.
These IDs apear also in the JSON [dictionary](https://github.com/ami-iit/yarp-openmct/blob/main/openmctStaticServer/dictionary.json) used in the **openmct static server**.
The idea was not really to have a nested structure "sens"->"imu" but more to have a composed name (like a flattened nested structure). In any case, if we come to the point where we rename the the telemetry data entries, it's better if we do it after I factorize the [dictionary](https://github.com/ami-iit/yarp-openmct/blob/main/openmctStaticServer/dictionary.json) and the [state](/blob/f2144093b7a6ea2b9fce1d0db7aab9a5bd33a63a/iCubTelemVizServer/icubtelemetry.js#L21). I mentioned in the call my intention to a single information source, and for instance generate the dictionary and state structures from that.

Reply:
I see. It makes sense! Since it was repeated I was assuming it could have been simply omitted in the file, but I see the rationale now.